CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Request PR review from any Apache Struts committer
+* @apache/struts-committers
```

Comment:
I think GitHub doesn't like the `*`

Reply:
Ah it might be complaining because it's a private team

Reply:
Right... not sure why this team is private as all committers are publicly known

Reply:
But where exactly are all struts-committers defined as a group on GitHub? I couldn't find any reference.

Reply:
I assume this is autogenerated group by ASF Infra base on the ASF records, yet I'm not sure which one is used for that as it doesn't include all the committers
https://people.apache.org/phonebook.html?unix=struts

Reply:
I see a list of 11 people - presumably the Struts committers who have made their Apache org membership public (private by default)?

Reply:
As far I understand you must link your GH account to the ASF account to be added to the team, yet still there is a difference.

Reply:
For reference https://issues.apache.org/jira/browse/INFRA-24905